rudder-core/src/test/scala/com/normation/rudder/migration/DBCommon.scala

```diff
@@ -63,13 +70,18 @@ import net.liftweb.common.Loggable
  */
 trait DBCommon extends Specification with Loggable with BeforeAfterAll {
 
+
   logger.info("""Set JAVA property 'test.postgres' to false to ignore that test, for example from maven with: mvn -DargLine="-Dtest.postgres=false" test""")
 
   val doDatabaseConnection = System.getProperty("test.postgres", "").toLowerCase match {
     case "true" | "1" => true
     case _ => false
   }
-  skipAllIf(!doDatabaseConnection)
+//  skipAllIf(!doDatabaseConnection)
+
+  // max time we wait for a request to complete
+  val MAX_TIME = Duration(300, TimeUnit.MILLISECONDS)
```

Comment:
300 millisecond is really short - on large instance, we can have multi-seconds queries ...